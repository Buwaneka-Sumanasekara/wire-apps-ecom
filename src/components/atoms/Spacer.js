import { View } from "react-native"

const Spacer = (props) => {
    const { size, isHeight } = props;
    return <View className={(isHeight ? `h-${size}` : `p-${size}`)} />
}

export default Spacer;