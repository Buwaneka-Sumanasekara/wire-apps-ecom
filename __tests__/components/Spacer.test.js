import React from 'react';
import renderer from 'react-test-renderer';
import Spacer from '../../src/components/atoms/Spacer';


describe('<Spacer/>', () => {
    it('rendered sucessfully', () => {
        const tree = renderer.create(<Spacer />).toJSON()
        expect(tree).toMatchSnapshot()
    });
});

