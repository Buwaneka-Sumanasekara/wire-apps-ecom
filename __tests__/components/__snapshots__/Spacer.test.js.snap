// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Spacer/> rendered sucessfully 1`] = `<View />`;
